addons/project_timesheet_holidays/models/hr_holidays.py

```diff
@@ -106,10 +106,26 @@ def _timesheet_prepare_line_values(self, index, work_hours_data, day_date, work_
             'company_id': self.holiday_status_id.timesheet_task_id.company_id.id or self.holiday_status_id.timesheet_project_id.company_id.id,
         }
 
+    def _check_missing_global_leave_timesheets(self):
+        for leave in self:
+            global_leaves = self.env['resource.calendar.leaves'].search([
+                ("resource_id", "=", False),
+                "|",
+                    "&",
+                        ("date_from", ">=", leave.date_from),
+                        ("date_from", "<=", leave.date_to),
+                    "&",
+                        ("date_to", ">=", leave.date_from),
+                        ("date_to", "<=", leave.date_to)
```

Comment:
Seems equivalent to.
```suggestion
                ("date_to", ">=", leave.date_from),
                ("date_from", "<=", leave.date_to),
```

Reply:
Good catch!